ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -428,6 +431,12 @@ open class SegmentedControl: UIControl {
         }
     }
 
+    private func updateAccessibilityHints() {
+        for (index, button) in buttons.enumerated() {
+            button.accessibilityHint = String(format: "Accessibility.MSPillButtonBar.Hint".localized, index + 1, items.count)
```

Comment:
do we need to use `localizedStringWithFormat` here instead?

Ref: [here](https://developer.apple.com/documentation/foundation/nsstring/1497301-localizedstringwithformat)